remoteit/desktop
# [DESK-542] Application level state

### Description


Moved page state in scanning and device setup to the application level, so that it's retained across page changes and should be protected from users accidentally or mistakenly seeing wrong state.

### Checklist




### Test plan



1. Sign in
2. Register a device, or use an already registered device
3. Add and remove services manually and with the network scan
4. Change pages while in progress
5. The state should be retained even when going to a new page and returning

### Ticket



<https://remoteit.atlassian.net/browse/DESK-542>
